returns/primitives/container.py

```diff
@@ -29,6 +29,13 @@ def __str__(self) -> str:
             str(self._inner_value),
         )
 
+    def __repr__(self) -> str:
```

Comment:
There's no need to add a new method, we can just rename `__str__` to `__repr__`

Reply:
Sure I will do that!